### PR TITLE
Adding name to HelloWorld Example

### PR DESCRIPTION
**Pull request by vncoelho**

### CODE REVIEW

File: templates/Template.CSharp/Contract1.cs [hunk 1]
```diff
@@ -3,9 +3,11 @@
 using Neo.SmartContract.Framework.Services.Neo;
 using System;
 using System.Numerics;
+using System.ComponentModel;
 
 namespace $safeprojectname$
 {
+    [DisplayName("Hello World Demo")]
```
Comment (shargon) [inline templates/Template.CSharp/Contract1.cs:10]: Just "Hello World"?

Comment (vncoelho) [reply templates/Template.CSharp/Contract1.cs:10]: Could be, I followed an oracle example I saw from @Tommo-L.

**Review by shargon — APPROVED** [review]

File: templates/Template.CSharp/Contract1.cs (hunk 1 above)
Comment (Tommo-L) [reply templates/Template.CSharp/Contract1.cs:10]: We moved oracle demo there now. https://github.com/neo-project/examples/pull/34/files

**Review by Tommo-L — APPROVED** [review]

**Comment by erikzhang** [comment]
This is not a demo. This is a **project template**. I will revert this commit.

**Pull request merged**